css/home-page.css

```diff
@@ -9,6 +9,7 @@
 @import './components/button.css';
 @import './components/contact.css';
 @import './components/creativity.css';
+@import './components/flaticon.css';
```

Comment:
Perkelti prie /* 3rd party */ dalies CSS faile.

Reply:
moved flaticon and fontawesome to 3rd party